AR Collaboration/Assets/Client/Scripts/CameraController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraController : MonoBehaviour
+{
+    public PlayerManager player;
+    public float sensitivity = 100f;
+    public float clampAngle = 85f;
+
+    private float verticalRotation;
+    private float horizontalRotation;
+
+    private void Start()
+    {
+        verticalRotation = transform.localEulerAngles.x;
+        horizontalRotation = player.transform.eulerAngles.y;
+    }
+
+    private void Update()
+    {
+        Look();
+        Debug.DrawRay(transform.position, transform.forward * 2, Color.red);
+    }
+
+    private void Look()
+    {
+        float _mouseVertical = -Input.GetAxis("Mouse Y");
```

Comment:
Will this be changed to Vive contoller input? or should it always be mouse input? (Mouse seems easier)